README.md

```diff
@@ -178,7 +178,7 @@ open("rust_g", O_RDONLY|O_NONBLOCK|O_LARGEFILE|O_DIRECTORY|O_CLOEXEC) = -1 ENOTD
 If you're still having problems, ask in the [Coderbus Discord]'s
 `#tooling-questions` channel.
 
-You can also try [tgstation]'s IRC, `#coderbus` on Rizon, but it is usually
+You can also try [/tg/station]'s IRC, `#coderbus` on Rizon, but it is usually
```

Comment:
Isn't the actual service dead?

Reply:
no

Reply:
![image](https://user-images.githubusercontent.com/4741640/180579844-cef3517c-a8de-40c3-be6a-829877bb6293.png)

Reply:
it is just bouncer people though

Reply:
Must have been down for maintenance last time I checked it then.

Reply:
https://github.com/tgstation/tgstation/pull/63207